acceptancetests/repository/charms/appdata-sink/hooks/source-relation-changed

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eux
+
+remote_app=${JUJU_REMOTE_APP:-}
+remote_unit=${JUJU_REMOTE_UNIT:-}
+if [[ -n $remote_app ]]; then
```

Comment:
So if I did it correctly JUJU_REMOTE_APP is *always* set, but JUJU_REMOTE_UNIT is set sometimes.
So I think we want
if [[ -n $remote_unit ]]; then

as the check, so that if there is a remote unit, then we treat it as a unit context, not "if there isn't an app", we treat it as a unit context. (Obviously if we change the if, we need to swap the content bodies)

Reply:
Ah right - I'll update this.